src/app.module.ts

```diff
@@ -1,28 +1,28 @@
-import { Module } from '@nestjs/common';
-import { TypeOrmModule } from '@nestjs/typeorm';
-import { AppController } from './app.controller';
-import { AppService } from './app.service';
-import { AuthModule } from './auth/auth.module';
-import { UserModule } from './user/user.module';
+import { Module } from "@nestjs/common";
+import { TypeOrmModule } from "@nestjs/typeorm";
+import {AuthModule} from "./auth/auth.module";
```

Comment:
Very minor, but inconsistent spacing here. Is there a linter rule for this actually?

Reply:
https://eslint.org/docs/rules/object-curly-spacing seems to be